src/components/application_manager/include/application_manager/message_helper.h

```diff
@@ -279,6 +284,20 @@ class MessageHelper {
   static smart_objects::SmartObjectList CreateAddCommandRequestToHMI(
       ApplicationConstSharedPtr app, ApplicationManager& app_mngr);
 
+  /**
+   * @brief Creates Message for sending to HMI for unsubscribe from specified
+   * VehicleData. Creates and returns message with vehicle data for unsubscribe.
+   * @param vehicle_data - collection with vehicle data types to ubsubscribe.
+   * @param app - application, that must will be unsubscribe from specified
```

Comment:
@IGapchuk `application which should be unsubscribed from specified vehicle data`